### PR TITLE
QC utilities for working with SimM

### PR DESCRIPTION
**Pull request by edsko**
Some QuickCheck utilities to work with `SimM`, analogues of existing standard functions for working directly with `ST`.

### CODE REVIEW

File: ouroboros-network-testing/src/Ouroboros/Network/Testing/QuickCheck.hs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+-- | 'SimM' analogue of 'runSTGen'
+--
+-- > runSTGen  :: (forall s. Gen (ST   s a)) -> Gen a
+-- > runSimGen :: (forall s. Gen (SimM s a)) -> Gen a
+runSimGen :: (forall s. Gen (SimM s a)) -> Gen a
+runSimGen f = do
+    Capture eval <- capture
+    return $ runSimOrThrow (eval f)
+
+-- | 'SimM; analogue of 'monadicST'
```
Comment (mrBliss) [inline ouroboros-network-testing/src/Ouroboros/Network/Testing/QuickCheck.hs]: Typo: replace the `;` with a `'`.

Comment (edsko) [reply ouroboros-network-testing/src/Ouroboros/Network/Testing/QuickCheck.hs]: Ah, yes. Will fix.

**Comment by edsko** [comment]
bors:

> :-1:  Rejected by too few approved reviews

:broken_heart:

**Pull request merged**